torchao/prototype/grouped_mm/__init__.py

```diff
@@ -0,0 +1,370 @@
+from typing import Optional, Tuple
```

Comment:
seems like folder name should be `grouped_scaled_mm` instead of `grouped_mm`?  Or do we plan to add a `grouped_mm` here too?

Reply:
Updated folder name